anecdotes/src/App.js

```diff
@@ -0,0 +1,35 @@
+import './App.css';
+import { useState } from 'react'
+
+const App = () => {
+  const anecdotes = [
+    'If it hurts, do it more often',
+    'Adding manpower to a late software project makes it later!',
+    'The first 90 percent of the code accounts for the first 10 percent of the development time...The remaining 10 percent of the code accounts for the other 90 percent of the development time.',
+    'Any fool can write code that a computer can understand. Good programmers write code that humans can understand.',
+    'Premature optimization is the root of all evil.',
+    'Debugging is twice as hard as writing the code in the first place. Therefore, if you write the code as cleverly as possible, you are, by definition, not smart enough to debug it.',
+    'Programming without an extremely heavy use of console.log is same as if a doctor would refuse to use x-rays or blood tests when diagnosing patients'
+  ]
+   
+  const [selected, setSelected] = useState(0)
+  function randomItem () {
+    const randomNumber = Math.floor(Math.random() * anecdotes.length);
+      setSelected(randomNumber);
+   console.log(selected)
```

Comment:
This won't log the correct number consistently as the `useState` function won't have updated the `selected` value.

This is the section in the docs that covers this - https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous